### ISSUE TITLE
Spacy import in v2.1 appears to depend on Japanese language class

### ISSUE THREAD

**Issue by jdukatz** [explosion/spaCy#3446]
I'm running spacy in a docker container, and to save on the install size, I'm removing any unused language classes a la this [issue](https://github.com/explosion/spaCy/issues/2851). However, with the upgrade to v2.1, importing spacy seems to fail specifically because the japanese language class is missing (deleting all but that class results in it working fine).

## How to reproduce the behaviour
 - Install spacy v2.1 and download the `en_core_web_sm` language model (haven't tested with other models but it seems to be a code issue, not a model issue).
 - Run `rm -rf [environment]/lib/python3.6/site-packages/spacy/lang/ja`
 - I see the following stacktrace when trying to import spacy:

```
$ python
Python 3.6.5 (default, Apr 25 2018, 14:23:58)
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import spacy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jack/src/swoop/swoop4X/python/env/lib/python3.6/site-packages/spacy/__init__.py", line 11, in <module>
    from .cli.info import info as cli_info
  File "/Users/jack/src/swoop/swoop4X/python/env/lib/python3.6/site-packages/spacy/cli/__init__.py", line 13, in <module>
    from .ud import ud_train, ud_evaluate  # noqa: F401
  File "/Users/jack/src/swoop/swoop4X/python/env/lib/python3.6/site-packages/spacy/cli/ud/__init__.py", line 2, in <module>
    from .ud_train import main as ud_train  # noqa: F401
  File "/Users/jack/src/swoop/swoop4X/python/env/lib/python3.6/site-packages/spacy/cli/ud/ud_train.py", line 33, in <module>
    from ...lang import ja
ImportError: cannot import name 'ja'
```

The workaround is obviously to just leave the ja module in, but I wanted to log an issue in case it's unintended behavior, since it seems like sort of an oddly specific dependency.

## Your Environment

* Operating System: macOS
* Python Version Used: 3.6
* spaCy Version Used: 2.1.0
* Environment Information: Happens in a python 3.6 virtualenv with spacy installed as well as in my docker container.

**Comment by ines** [comment]
Could you try and updrade to `2.1.1`? This should have been fixed by #3441!

**Comment by jdukatz** [comment]
Ha, good timing. Yep, that fixed it. Thanks!
